libraries/botbuilder-dialogs-adaptive/src/templates/activityTemplate.ts

```diff
@@ -60,19 +61,18 @@ export class ActivityTemplate
      */
     public async bind(dialogContext: DialogContext, data: DialogStateManager): Promise<Partial<Activity>> {
         if (this.template) {
-            const languageGenerator = dialogContext.services.get(languageGeneratorKey) as LanguageGenerator<
-                Partial<Activity>,
-                DialogStateManager
+            let languageGenerator = dialogContext.services.get(languageGeneratorKey) as LanguageGenerator<
+                unknown,
+                DialogStateManager | Record<string, unknown>
             >;
 
-            if (languageGenerator) {
-                const lgResult = await languageGenerator.generate(dialogContext, this.template, data);
-                const result = ActivityFactory.fromObject(lgResult);
-                return Promise.resolve(result);
-            } else {
-                const message = MessageFactory.text(this.template, this.template);
-                return Promise.resolve(message);
+            if (languageGenerator == null) {
```

Comment:
You can one line this as `const languageGenerator = ... ?? new TemplateEngineLanguageGenerator()` or, if you want to keep it on two lines for readability, `languageGenerator ??= new TemplateEngineLanguageGenerator()`